.github/workflow-templates/install-frame-omni-bencher/action.yml

```
@@ -0,0 +1,11 @@
+name: Install frame-omni-bencher
+description: |
+  Install frame-omni-bencher
+
+runs:
+  using: "composite"
+  steps:
+    - name: Install frame-omni-bencher
+      shell: bash
+      run: |
+        cargo install frame-omni-bencher --locked
```

This defaults to the latest version so it will break our CI when a new version is released. Prefer to pin the version, ideally the same as in our polkadot-sdk fork branch.

```suggestion
        cargo install frame-omni-bencher --locked --git https://github.com/moondance-labs/polkadot-sdk --branch tanssi-polkadot-stable2506
```

If that doesn't compile then just pin to a specific version from crates io.